test/spmd/tensor/test_pointwise_ops.py

```diff
@@ -90,6 +90,35 @@ def test_dropout_errors(self):
                 torch.nn.functional.dropout,
             )
 
+    @with_comms
+    def test_mul(self):
+        device_mesh = DeviceMesh(self.device_type, list(range(self.world_size)))
+        torch.manual_seed(self.rank)
+        spec = [Replicate()]
```

Comment:
can we test both replicate and shard?

Reply:
I am testing Shard instead of Replicate since that is more relevant for the out variant.